src/tutorials/0007-verified-storage-on-filecoin/02.md

```diff
@@ -0,0 +1,23 @@
+---
+    title: "Proof-of-Replication (PoRep)"
+    type: "multiple-choice" # can be "text", "code", "file-upload" or "multiple-choice"
+---
+
+In a Proof of Replication, a storage miner proves that they are storing a _physically_ unique copy, or _replica_, of the data. This process happens just once, at the time the data is first stored by the miner and sealed into a sector.
+
+After a client negotiates a storage deal with a miner, the data transfer process begins. A client hires a miner to store a _piece_ of data, which might be a whole or partial file. Miners store these pieces from one or more clients in _sectors_, the fundamental storage unit used by Filecoin.
+
+## The pre-commit phase: Replication & CommD
+The replication of data happens in a _pre-commit_ phase, in which the storage miner places the replicated piece data into a sector. Each piece of data stored in Filecoin has a _payload CID_, similar to an IPFS CID, which is a cryptographic hash of the data itself. Since these CIDs are unique, identical CIDs guarantee identical content. No matter where data is stored or who's storing it, identical data will produce identical payload CIDs.
```

Comment:
re: "similar to an IPFS CID", I confirmed that given the same content to be stored/pinned, the _payload CID_ is identical to the IPFS CID. Would be good to clarify that similarity.
https://filecoinproject.slack.com/archives/CHMNDCK9P/p1591642395375000.

Reply:
Would changing "similar to an IPFS CID" to "identical to an IPFS CID cover it, or is that missing necessary nuance? Do you want me to go into the chunking caveat that Hannah mentioned in that Slack thread?

Reply:
yup, I think that would cover it!